logging-api/src/main/java/com/cisco/oss/foundation/logging/transactions/SchedulerLogger.java

```diff
@@ -47,6 +56,20 @@ public static void success(String schedulerResult) {
 
     }
 
+    public static void successAsync(SchedulerLogger schedulerLogger) {
+
+        FlowContextFactory.deserializeNativeFlowContext(TransactionLogger.getFlowContextAsync(schedulerLogger));
```

Comment:
how about:
FlowContextFactory.deserializeNativeFlowContext(flowContext)

This does what you want but uses the internally stored flowContext as it is protected in parent class. Would that work?